test/k8sT/manifests/demo_ds_dualstack.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: testds-service-ipv6
+spec:
+  ipFamilies:
+  - IPv6
```

Comment:
The file name might be a little bit confused if anyone didn't dig deeper for context (e.g. the comment in 1.19/demo_dualstack.yaml file). Basically, the manifest is _only_ for ipv6 service, but not for _dual stack_.

In my opinion, these files could be named as `*_ipv6.yaml` 🤔

Reply:
Good point. While for now the manifests only contain IPv6 services, going forward we might add some tests which are not IPv6 specific but rather target specific dual-stack features. From the top of my head, one such feature I can think of is to validate if Cilium honors the` ipFamilyPolicy` field(I will be working on this next). 
Happy to change the name of the files for now though. Thoughts @christarazi  @aanm ?

Reply:
It's fine, the overall intent of the filename is to immediately tell to what it relates to.